servant-client/servant-client.cabal

```diff
@@ -41,7 +41,7 @@ library
     , base64-bytestring     >= 1.0.0.1  && < 1.1
     , bytestring            >= 0.10     && < 0.11
     , exceptions            >= 0.8      && < 0.9
-    , http-api-data         >= 0.1      && < 0.3
+    , http-api-data         >= 0.1      && < 0.4
```

Comment:
You should bump the lower bound to `>= 0.3`. Or use CPP around `import Web.FormUrlEncoded`.
I'd prefer the former.